myapp/apps.py

```diff
@@ -1,5 +1,14 @@
 from django.apps import AppConfig
-
+import sys
 
 class MyappConfig(AppConfig):
     name = 'myapp'
+
+    def ready(self):
+        if 'runserver' not in sys.argv:

```

Comment:
a better place to start the thread is in `commands/stock_manager.py`, to be run whenever the command "runserver" provided, like this you get:
1) a more logical place to search for the thread initialization.
2) no need for hard-coded condition (`if 'runserver' not in sys.argv`)
3) more reliability on starting the service.  

it's just a refactoring thing, you can keep it as it is if you want